design-proposals/klip-62-ksqldb-ruby-client.md

```diff
@@ -0,0 +1,53 @@
+# KLIP 62 - ksqlDB Ruby Client
+
+**Author**: Lapo Elisacci (@LapoElisacci) | 
+**Release Target**: TBD | 
+**Status**: _In Discussion_ | 
+**Discussion**: TBD
+
+**tl;dr:** Develop a Ruby HTTP2 Client to request ksqlDB REST API
+
+## Motivation and background
+
+Allow Ruby / Ruby on Rails developers to get started with ksqlDB.
+
+## What is in scope
+
+* A lightweight but complete and configurable Ruby client.
```

Comment:
It will be helpful to point out what we mean by "configurable" here. What level of configurability do we want to provide here? We can just copy the level of configurability provided by the java client